pkg/ksym/ksym.go

```diff
@@ -56,6 +56,7 @@ func NewKsymCache(logger log.Logger) *Cache {
 	}
 }
 
+// TODO(kakkoyun): https://github.com/aquasecurity/libbpfgo/blob/main/helpers/kernel_symbols.go
```

Comment:
that looks better but ultimately we should probably just implement a real parser